cukes-rest/src/main/java/lv/ctco/cukesrest/internal/GuiceModule.java

```diff
@@ -7,6 +7,7 @@
 import lv.ctco.cukesrest.internal.context.*;
 import lv.ctco.cukesrest.internal.switches.*;
 import org.aopalliance.intercept.*;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
I've removed dependency to Apache Commons to avoid transitive version problems. There is now `lv.ctco.cukesrest.internal.helpers.Strings` for this kind of things